katello/backup.rb

```diff
@@ -67,13 +68,19 @@ def configure_configs
         '/var/lib/puppet/ssl'
       ]
 
+      scenario_answers = load_scenario_answers(@last_scenario)
+
       katello_configs = [
         "/etc/candlepin",
         "/etc/foreman",
         "/etc/hammer",
         "/etc/sysconfig/tomcat*",
         "/etc/tomcat*",
         "/var/lib/candlepin",
+	scenario_answers["certs"]["server_cert"],
+	scenario_answers["certs"]["server_key"],
+	scenario_answers["certs"]["server_cert_req"],
+	scenario_answers["certs"]["server_ca_cert"]
```

Comment:
is it just me or is the alignment off ?

Reply:
:+1: great catch, fixing now

Reply:
Actually @sean797 it's correct, just looks strange here. If you look at "raw" code without the markup, you'll see it is aligned correctly.

Reply:
Its because you've used a tab 😈 please uses spaces